packages/SketchMorph2-Core.package/M2GripMorph.class/instance/initialize.st

```diff
@@ -0,0 +1,8 @@
+initialization
+initialize
+	super initialize.
+	self
+		showHandle: true;
+		color: Color green;
```

Comment:
Needs to be updated before merging.

Reply:
Adjusted in [1802a96](https://github.com/hpi-swa-teaching/SketchMorph2/pull/48/commits/1802a96f3eb74a27efb63c387e2fb00b2f692b1d).